pkg/network/kubemacpool.go

```diff
@@ -84,6 +71,28 @@ func renderKubeMacPool(conf *opv1alpha1.NetworkAddonsConfigSpec, manifestDir str
 	return objs, nil
 }
 
+func fillDefaultsKubeMacPool(conf, previous *opv1alpha1.NetworkAddonsConfigSpec) error {
+	if conf.KubeMacPool.RangeStart == "" || conf.KubeMacPool.RangeEnd == "" {
```

Comment:
please add an inline comment saying that if user hasn't explicitly requested a range, we try to reuse previously applied